ykman-gui/qml/YubiKey.qml

```diff
@@ -436,4 +436,13 @@ Python {
     function pivReadCertificate(slot, cb) {
         doCall('yubikey.controller.piv_read_certificate', [slot], cb)
     }
+
+    function pivImportFile(slot, fileUrl, password, pin, managementKey, cb) {
+        doCall('yubikey.controller.piv_import_file',
+               [slot, fileUrl, password, pin, managementKey], cb)
+    }
+
+    function pivCanParse(fileUrl, cb) {
+        doCall('yubikey.controller.piv_can_parse', [fileUrl], cb)
```

Comment:
Let's use `doPivCall` here after merging (`doPivCall` is in the `piv` branch but not this one).

Reply:
Merged piv into PR, updated calls.